libs/sdk-ui-kit/src/StylingPicker/StylingPicker.tsx

```diff
@@ -0,0 +1,196 @@
+// (C) 2022 GoodData Corporation
+
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { injectIntl, WrappedComponentProps } from "react-intl";
+import cx from "classnames";
+import { areObjRefsEqual, ObjRef } from "@gooddata/sdk-model";
+import { IntlWrapper } from "@gooddata/sdk-ui";
+import { Bubble, BubbleHoverTrigger } from "../Bubble";
+import { Typography } from "../Typography";
+import { ContentDivider, DialogListHeader } from "../Dialog";
+import { Button } from "../Button";
+import { StylingPickerList } from "./StylingPickerList";
+import { StylingPickerItem, StylingPickerListItem } from "./StylingPickerListItem";
+import { DialogListLoading } from "../Dialog/DialogList/DialogListLoading";
+import { useMediaQuery } from "../responsive/useMediaQuery";
+import { Message } from "../Messages";
+import { IAlignPoint } from "../typings/positioning";
+
+const TOOLTIP_ALIGN_POINTS: IAlignPoint[] = [{ align: "bl tl", offset: { x: 10, y: 4 } }];
+
+/**
+ * @internal
+ */
+export interface IStylingPickerProps {
+    title: string;
+    basicItem: StylingPickerItem;
```

Comment:
should this be named `defaultItem`?
Or what about including all the items in `items` prop and then provide check for `isDefaultItem: (StylingPickerItem) => boolean`